pages/index.tsx

```diff
@@ -31,7 +32,15 @@ const Home: WithPageLayout = () => {
       }
 
       if (currentUser?.data?.session && onboarded === true) {
-        redirect("/hub/insights");
+        const redirectUrl = sessionStorage.getItem("redirectUrl");
```

Comment:
Would prefer to use the `redirectTo` from the auth provider to go directly back to the referring page after login

Reply:
Same conversation as the other `redirectTo` conversation: it doesn't work so this is a workaround.

Reply:
unfortunately, this won't work without the `redirectTo` doing the job in both instances :<